### PR TITLE
test pull request

### PR DESCRIPTION
**Pull request by sarathecoubian**
:)

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint platform, permissions, configuring
 
-Use *Anypoint Access Management* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
+Use *Access Management* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```
Comment (sarathecoubian) [inline modules/ROOT/pages/index.adoc:7]:
```suggestion
Use *Access Manager* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```
```suggestion
Use *Access Management* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint platform, permissions, configuring
 
-Use *Anypoint Access Management* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
+Use *Access Manager* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```
Comment (sarathecoubian) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Use *Access Management* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```
```suggestion
Use *Access Manager* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```

**Review by LizThomerson — APPROVED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (LizThomerson) [inline modules/ROOT/pages/index.adoc:7]: Thanks for the help!

Comment (LizThomerson) [reply modules/ROOT/pages/index.adoc:7]: Testing

Comment (LizThomerson) [reply modules/ROOT/pages/index.adoc:7]:
```suggestion
Use *Anypoint Access Management* to xref:managing-your-account.adoc#how-to-create-your-account[create your Anypoint Platform account] or configure federated xref:external-identity.adoc[External Identity].
```

**Pull request closed without merging**